vagrant-serverspec.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |gem|
   #add onlly dependencies for winrm without nokogiri
   gem.add_runtime_dependency 'winrm', '~> 2.1', '>= 2.1'
   gem.add_runtime_dependency 'os', '~> 0.9.6'
+  gem.add_runtime_dependency 'rspec_html_formatter', '~> 0.3', '>= 0.3.1'
```

Comment:
I guess if `rspec_html_formatter` is part of the rspec project then it should be Ok. Adding another dependency means more testing but I kind of understand the needed functionality. 

https://github.com/kingsleyh/rspec_reports_formatter (Kind of old). Do you have any sample HTML to take a look at the benefit of using this dependency?